tests/api.c

```diff
@@ -2627,12 +2627,12 @@ static int nonblocking_accept_read(void* args, WOLFSSL* ssl, SOCKET_T* sockfd)
 
 
 #if defined(HAVE_SESSION_TICKET) && \
-    ((defined(HAVE_CHACHA) && defined(HAVE_POLY1305)) || defined(HAVE_AESGCM)) && \
+  (/*(defined(HAVE_CHACHA) && defined(HAVE_POLY1305)) || */defined(HAVE_AESGCM)) && \
```

Comment:
No need to keep this around if we are not adding support for this in the example callback.

Reply:
Done. Added a `TODO` comment